umad/src/main/resources/application.conf

```diff
@@ -48,39 +49,7 @@ umad: {
       {
         class: "scala\\.tools\\.nsc\\.Global"
         method: "reporter_\\$eq"
-      }, // Var on traits get coverted into messy method calls chains - maybe we need to look at it at some poitn
```

Comment:
Random question: Where is best place to keep ideas about next perf improvements? I've got a feeling that `invokeinverface`s on var/vas may be really costly...

Reply:
`scalac_perf` issue tracker I think.